### PR TITLE
Created EntityFakers

### PR DESCRIPTION
**Pull request by Habiba67y**
EntityFakers static class created

### CODE REVIEW

File: src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Persistence/SeedData/EntityFakers.cs [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+using Training.TruckWorld.Backend.Persistence.DataContexts;
+
+namespace Training.TruckWorld.Backend.Persistence.SeedData;
+
+internal static class EntityFakers
+{
+    internal static Faker<User> GetUserFaker(IDataContext context)
+    {
+        return new Faker<User>()
+            .RuleFor(user => user.Id, Guid.NewGuid())
```
Comment (azizbek-abdurahmonov) [inline src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Persistence/SeedData/EntityFakers.cs]: Guid.NewGuid() - qavs bo'lishi kerak emas

File: src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Persistence/SeedData/EntityFakers.cs [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+            .RuleFor(user => user.FirstName, faker => faker.Person.FirstName)
+            .RuleFor(user => user.LastName, faker => faker.Person.LastName)
+            .RuleFor(user => user.EmailAddress, faker => faker.Person.Email);
+    }
+    internal static Faker<UserCredentials> GetUserCredentialsFaker(IDataContext context)
+    {
+        var usersId = new Stack<Guid>(context.Users.Select(user => user.Id));
+        return new Faker<UserCredentials>()
+            .RuleFor(userCredentials => userCredentials.Id, Guid.NewGuid())
```
Comment (azizbek-abdurahmonov) [inline src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Persistence/SeedData/EntityFakers.cs:22]: bu yerda ham ID generate qilishda () keremas

**Review by azizbek-abdurahmonov — DISMISSED** [review]

File: src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Persistence/SeedData/EntityFakers.cs (hunk 1 above)
Comment (Habiba67y) [reply src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Persistence/SeedData/EntityFakers.cs]: aha tushunarli

File: src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Persistence/SeedData/EntityFakers.cs (hunk 2 above)
Comment (Habiba67y) [reply src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Persistence/SeedData/EntityFakers.cs:22]: ok

**Review by azizbek-abdurahmonov — APPROVED** [review]
good

**Review by azizbek-abdurahmonov — APPROVED** [review]

**Review by botirjonabdumannonovv — APPROVED** [review]

**Pull request merged**